docs/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Building .NET Core Docker Images
+description: Understading Docker images and .NET Core
+keywords: .NET, .NET Core, Docker
+author: spboyer
+manager: wpickett
+ms.date: 08/29/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 03c28597-7e73-46d6-a9c3-f9cb55642739
+---
+
+
+#Building Docker Images for .NET Core Applications
+
+In order to get an understanding of how to use .NET Core and Docker together, we must first get to know the different Docker images that are offered and when is the right use case for them. Here we will walk through the variatons offered, build an ASP.NET Core Web API, use the Yeoman Docker tools to create a debuggable container as well as peek at how Visual Studio Code can assist in the process. 
+
+## Docker image variatons
+
+The `microsoft/dotnet` images are offerred in three different variatons, each for a specific use case.
+
+- `microsoft/dotnet:<version>-sdk` : i.e. **microsoft/dotnet-preview2-sdk**, this image contains the .NET Core SDK which includes the .NET Core and Command Line Tools (CLI). Use this image for your development processes; dev, test debug.
```

Comment:
no i.e., e.g. on the topics... change to *that is*
Missing punctuation on the end of the sentence: dev, test debug? I'd also avoid using short names like dev because it makes it hard to localize later on.